.github/workflows/build-test-lint.yml

```diff
@@ -0,0 +1,27 @@
+
+name: Build, Test, Lint for Pull Requests
+
+on: [pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
+
+      - name: Install
+        run: yarn install
+
+      - name: Build
+        run: yarn build
+
+      - name: Test
+        run: yarn test --watchAll=false 
+
+      - name: Lint
```

Comment:
Probably doesn't matter for the size of the codebase, but I generally move lint to the top. Linting fails relatively fast, I have test suites that can take over 15 minutes to run :)